SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -242,9 +242,17 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	TEnumAsByte<EServicesRegion::Type> ServicesRegion;
 
 	/** Controls the verbosity of worker logs which are sent to SpatialOS. These logs will appear in the Spatial Output and launch.log */
-	UPROPERTY(EditAnywhere, config, Category = "Logging", meta = (DisplayName = "Worker Log Level"))
+	UPROPERTY(config, meta = (DisplayName = "Worker Log Level"))
```

Comment:
Worth adding a comment here about deprecation and adding a UNR ticket to remove it after the 0.12 release (also reference it's part of an settings change upgrade path)

Reply:
Done!